egs2/librispeech/hubert_asr1/hubert_asr.sh

```diff
@@ -0,0 +1,552 @@
+lm_config=        # Config for language model training.
+lm_args=          # Arguments for language model training, e.g., "--max_epoch 10".
+                  # Note that it will overwrite args in lm config.
+use_word_lm=false # Whether to use word language model.
+num_splits_lm=1   # Number of splitting for lm corpus.
+# shellcheck disable=SC2034
+word_vocab_size=10000 # Size of word vocabulary.
+
+# ASR model related
+num_splits_asr=1           # Number of splitting for lm corpus.
+
+# Decoding related
+inference_tag=    # Suffix to the result dir for decoding.
+inference_config= # Config for decoding.
+inference_args=   # Arguments for decoding, e.g., "--lm_weight 0.1".
+                  # Note that it will overwrite args in inference config.
+inference_lm=valid.loss.ave.pth       # Language modle path for decoding.
+inference_asr_model=valid.acc.best.pth # ASR model path for decoding.
+                                      # e.g.
+                                      # inference_asr_model=train.loss.best.pth
+                                      # inference_asr_model=3epoch.pth
+                                      # inference_asr_model=valid.acc.best.pth
+                                      # inference_asr_model=valid.loss.ave.pth
+download_model= # Download a model from Model Zoo and use it for decoding.
+
+# Pretrain related
+pretrain_train_set=
+pretrain_valid_set=
+pretrain_config_list=
+n_clusters_list=
+feature_list=
+
+# Finetuen related
+finetune_train_set=
+finetune_valid_set=
+finetune_test_sets=
+finetune_config=
+
+
+bpe_train_text=  # Text file path of bpe training set.
+lm_train_text=   # Text file path of language model training set.
+lm_dev_text=     # Text file path of language model development set.
+lm_test_text=    # Text file path of language model evaluation set.
+nlsyms_txt=none  # Non-linguistic symbol list if existing.
+cleaner=none     # Text cleaner.
+g2p=none         # g2p method (needed if token_type=phn).
+lang=noinfo      # The language type of corpus.
+score_opts=                # The options given to sclite scoring
+local_score_opts=          # The options given to local/score.sh.
+asr_speech_fold_length=800 # fold_length for speech data during ASR training.
+asr_text_fold_length=150   # fold_length for text data during ASR training.
+lm_fold_length=150         # fold_length for LM training.
+
+help_message=$(cat << EOF
+Usage: $0 --finetune_train-set "<finetune_train_set_name>" --finetune_valid-set "<finetune_valid_set_name>" --finetune_test_sets "<finetune_test_set_names>"
+
+Options:
+    # General configuration
+    --stage                # Processes starts from the specified stage (default="${stage}").
+    --stop_stage     	   # Processes is stopped at the specified stage (default="${stop_stage}").
+    --pretrain_start_iter  # Pretrain starts from the specified iteration (0 mean MFCC iteraion)
+    --pretrain_stop_iter   # Pretrain is stopped from the specified iteration (0 mean MFCC iteraion)
```

Comment:
It is better to follow the format.
```suggestion
    --pretrain_start_iter  # Pretrain starts from the specified iteration (0 mean MFCC iteraion, default="${pretrain_start_iter}").
    --pretrain_stop_iter   # Pretrain is stopped from the specified iteration (0 mean MFCC iteraion, default="${pretrain_stop_iter}").
```